examples/mini-app/src/components/game/DiceGame.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, ChangeEvent } from 'react';
+import { Card, CardContent, CardFooter } from '../ui/card';
+import { Input } from '../ui/input';
+import { Button } from '../ui/button';
+import { Slider } from '../ui/slider';
+import { Popover, PopoverContent, PopoverTrigger } from '../ui/popover';
+import { Info } from 'lucide-react';
+import { cn } from '../../lib/utils';
+
+export interface DiceGameProps extends React.HTMLAttributes<HTMLDivElement> {
+  theme?: 'light' | 'dark' | 'system';
+  customTheme?: React.CSSProperties;
+}
+
+export function DiceGame({
+  theme = 'system',
+  customTheme,
+  className,
+  ...props
+}: DiceGameProps) {
+  const [sliderValue, setSliderValue] = useState([25]);
+  const [betAmount, setBetAmount] = useState('1');
+
+  const multiplier = (99 / (99 - sliderValue[0])).toFixed(2);
+  const targetPayout = (parseFloat(betAmount || '0') * parseFloat(multiplier)).toFixed(2);
+  const winChance = (99 - sliderValue[0]);
```

Comment:
Name the constants please. 

Can affiliates change 99 to something else in game contract settings?